menus/EscMenu.gd

```diff
@@ -0,0 +1,20 @@
+extends Control
+
+func _ready():
+	set_process_input(true)
+
+func _input(event):
+	# react to the ESC key
+	if(event.type==InputEvent.KEY):
+		if(event.pressed==true && event.is_action_pressed("ui_cancel")):
```

Comment:
`event.is_action_pressed("ui_cancel")` would be enough to remove all checks here. (what if I use cancel from a joystick...)